gitops/README.md

```diff
@@ -128,7 +132,7 @@ ssh-keyscan github.com > ssh/known_hosts
 ```console
 echo "Host github.com
   HostName github.com
-  IdentityFile ~/.ssh/id_rsa" > ssh/config
+  IdentityFile ~/ssh/id_rsa" > ssh/config
```

Comment:
So I was a bit confused because from the prior steps, it creates the `rsa` key in the `ssh` folder and not the `.ssh ` folder. Shouldn't the IdentityFile then point to `~/ssh/id_rsa` instead? I might be misunderstanding the flow.

Reply:
Based on [this](https://docs.openshift.com/pipelines/1.15/secure/authenticating-pipelines-repos-using-secrets.html#op-configuring-git-ssh-authentication-using-workspaces_authenticating-pipelines-repos-using-secrets) it seems that `.ssh` is expected.

This secret is being used by [tekton pipeline](https://github.com/parodos-dev/orchestrator-helm-chart/blob/b2a48b36ddab46b8e2369d7e3695477ee5b07b41/charts/orchestrator/templates/tekton-tasks.yaml#L39) to build the workflow from its github repository.
Looking at the tekton run logs it looks like this:
```
step-git

+ '[' false '=' true ]
+ '[' true '=' true ]
+ cp -R /workspace/ssh-directory /home/git/.ssh
```
therefore, it should remain `.ssh` since it doesn't refer to the local directory, but to the destination one.

Reply:
Ah, got it. My bad, I thought it referred to the local directory. Thanks for the clarification